src/components/FieldGuideItemDetail.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import {
+  ScrollView,
+  View
+} from 'react-native'
+import EStyleSheet from 'react-native-extended-stylesheet'
+import StyledMarkdown from './StyledMarkdown'
+import StyledText from './StyledText'
+import SizedImage from './SizedImage'
+import Button from './Button'
+
+const ITEM_ICON_HEIGHT = 100
```

Comment:
Changing this to `ITEM_ICON_RADIUS` might make more sense for when it is used below in the styles.